website/docs/r/databricks_workspace.html.markdown

```diff
@@ -50,19 +50,23 @@ The following arguments are supported:
 
 ~> **NOTE** Downgrading to a `trial sku` from a `standard` or `premium sku` will force a new resource to be created.
 
-* `managed_services_cmk_key_vault_key_id` - (Optional) Customer managed encryption properties for the Databricks Workspace managed resources(e.g. Notebooks and Artifacts). Changing this forces a new resource to be created.
+* `managed_services_cmk_key_vault_key_id` - (Optional) Customer managed encryption properties for the Databricks Workspace managed resources(e.g. Notebooks and Artifacts).
+
+* `managed_disk_cmk_key_vault_key_id` - (Optional) Customer managed encryption properties for the Databricks Workspace managed disks. Changing this forces a new resource to be created.
```

Comment:
Ok, so for the `managed_disk_cmk...` I've removed both `ForceNew` and docs. But as to `no_public_ip` - I'd leave it on `ForceNew`, since the use-case where it can be updated is somewhat all too narrow.

@stephybun What do you think?
@alexott Also, can you please let @stephybun know what needs to be done to the subscription by MSFT to allow for public_network_access updates? Thanks!

Reply:
`ForceNew` on `no_public_ip` unfortunately may block some of the old customers from upgrade... I don't see the problem of failing if the people are trying to update it, instead of forcing recreation

Reply:
I've updated the `no_public_ip` parameter to not force re-creation and added a note in the docs describing which use-case allows for an update.

Reply:
I think we should add a `ForceNewIfChange` to recreate the workspace if a user tries to update from `true` to `false`.

Reply:
@stephybun we need to check with dev team if it's really required. I expect that it should be possible to update it in-place